src/components/SortControl/SortControl.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import './SortControl.css';
+import Option from '../Option/Option';
+
+interface SortControlProps {
+	initialValue: string;
+	onChange: (value: string) => void;
+}
+
+const options = ['releaseDate', 'title'];
```

Comment:
What is the reason to use camel case options? I guess you can use readable strings here